### ISSUE TITLE
Cannot build Kessler CCPP cap using "feature/capgen" ccpp-framework branch.

### ISSUE THREAD

**Issue by nusbaume** [NCAR/atmospheric_physics#13]
When using the "feature/capgen" branch of the ccpp-framework to try and generate a CCPP cap with the Kessler suite, the generation fails with the following error:

> parse_source.ParseInternalError: Attempt to set intent of temp, at /glade/work/nusbaume/SE_projects/new_cam_sandbox/CAMDEN/src/physics/ncar_ccpp/state_converters.meta:18 to in, only 'inout' allowed.
>intent source: temp, at /glade/work/nusbaume/SE_projects/new_cam_sandbox/CAMDEN/src/physics/ncar_ccpp/kessler_update.meta:76

This occurs both when trying to build CAMDEN with the Kessler suite, and when trying to run the capgen script (`ccpp_capgen.py`) independently.  The original "CPF_0.2.016" tag used in CAMDEN was able to generate the caps without an issue, so something must have changed on the ccpp-framework side to cause this problem.

Finally, I should note that even though I am posting this in atmospheric_physics, it is unclear to me if this is actually a problem with the current CCPP-ized physics routines, or if it is a bug in the feature/capgen branch of the ccpp-framework.  However, given that I can't just walk down the hall and ask, this seemed like as reasonable a place as any to bring it up.

Any thoughts or ideas would certainly be appreciated!

**Comment by gold2718** [comment]
Can you provide version information for CAMDEN? I tried this and got:
```
parse_source.CCPPError: Input argument for kessler_run, geopotential_height, not found.
```

**Comment by nusbaume** [comment]
I am using the NCAR/CAMDEN master branch, with the last commit being:

---------------------------------------------------------------
commit 88aa061155430bc1247d1d80261e9cc0e9a07eeb
Merge: 75b0e65 8a0771a
Author: cacraigucar <cacraig@ucar.edu>
Date:   Tue Mar 17 13:05:51 2020 -0600

Merge pull request #27 from cacraigucar/master

fix when running with single node

---------------------------------------------------------------

I don't know if this matters for your case, but I also had to run manage_externals with:

```
[ncar-physics]
local_path = src/physics/ncar_ccpp
protocol = git
repo_url = https://github.com/NCAR/atmospheric_physics
tag = version0_00_002
required = True
```

present in `Externals_CAM.cfg`.

**Comment by nusbaume** [comment]
I just tried it with atmospheric_physics tag `version0_00_004` and got the same error you did.  I can try and debug that and see if the original error I was getting shows up once this new bug is fixed.

**Comment by nusbaume** [comment]
It looks like the `geopotential_height` error is caused by a mis-labeling in CAMDEN's registry.  I have attached a fixed version of the registry (link at the bottom with gzipped file), which now returns the original error that I described above (and that I am not quite sure how to solve at the moment).

Also, I have created a new issue in the CAMDEN repo describing this bug in the registry.  I can either make a PR now to fix it, or I can wait until we know that we have a fully working version with the new framework code, and then make a PR that also includes any other bug fixes we possibly find.  Just let me know if there is a preference!

[registry.xml.gz](https://github.com/NCAR/atmospheric_physics/files/4424247/registry.xml.gz)
